src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/PostResourceAllocationAction.java

```diff
@@ -43,11 +44,25 @@ public PostResourceAllocationAction(PersistenceManager persistenceManager) {
         Flow flow = getFlow(flowId);
         Message message = buildResponseMessage(flow, stateMachine.getCommandContext());
         stateMachine.setOperationResultMessage(message);
+
+        // Notify about successful allocation.
+        stateMachine.getEventListeners().forEach(listener -> listener.onResourcesAllocated(flowId));
```

Comment:
Please, use "notifyEventListeners" method available in FlowProcessingWithEventSupportFsm class.

Reply:
Fixed.